certified-connectors/personr/apiDefinition.swagger.json

```diff
@@ -0,0 +1,492 @@
+                                        "nameFirst": {
+                                            "type": "string",
+                                            "description": "nameFirst"
+                                        },
+                                        "nameLast": {
+                                            "type": "string",
+                                            "description": "nameLast"
+                                        },
+                                        "tags": {
+                                            "type": "array",
+                                            "items": {},
+                                            "description": "tags"
+                                        },
+                                        "flowName": {
+                                            "type": "string",
+                                            "description": "flowName"
+                                        },
+                                        "status": {
+                                            "type": "string",
+                                            "description": "status"
+                                        }
+                                    },
+                                    "description": "response"
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+        },
+        "/api-request-applicant-check": {
+            "post": {
+                "responses": {
+                    "default": {
+                        "description": "default",
+                        "schema": {}
+                    }
+                },
+                "summary": "Request Applicant Check",
+                "description": "Confirm all documents are uploaded, and start the check.",
+                "operationId": "Api-request-applicant-check",
+                "parameters": [
+                    {
+                        "name": "body",
+                        "in": "body",
+                        "required": false,
+                        "schema": {
+                            "type": "object",
+                            "properties": {
+                                "applicantId": {
+                                    "type": "string",
+                                    "description": "applicantId",
+                                    "title": "Applicant ID"
+                                }
+                            }
+                        }
+                    }
+                ]
+            }
+        }
+    },
+    "definitions": {},
+    "parameters": {},
+    "responses": {},
+    "securityDefinitions": {
+        "Auth. Token": {
+            "type": "apiKey",
+            "in": "header",
+            "name": "Authorization"
+        }
+    },
+    "security": [
+        {
+            "Auth. Token": []
+        }
+    ],
+    "tags": []
+}
```

Comment:
"x-ms-connector-metadata" this tag is missing, could you please add the same here, please refer the template of swagger.json file. https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/apiDefinition.swagger.json